source/css/_common/components/post/post-followme.styl

```diff
@@ -0,0 +1,60 @@
+.followme {
```

Comment:
I'd suggest someone to refine this piece of css. I'm no css expert so this may not be the best practise.

Reply:
@stevenjoezhang need u help XD 
not a css expert too

Reply:
Of course, I would be happy to help with this CSS work 🤪